fast-setup.sh

```diff
@@ -0,0 +1,10 @@
+git submodule update --init --recursive --depth 1
+echo "please read this file for further instructions"
+
+### INSTRUCTIONS
+# make -j 12 # change 12 to desired parallelism
+### find port in command from make # 9821
```

Comment:
Is 9821 just an example? Can you make the port number some placeholder e.g., `<PORT_NUMBER>`?